xc/priority_auths.lua

```diff
@@ -8,40 +8,49 @@ local _M = { }
 -- @return true if authorized, false if denied, nil if unknown
 -- @return reason why the authorization is denied (optional, required only when denied)
 function _M.authorize(service_id, credentials, metric)
-  local redis_pub, ok_pub = redis_pool.acquire()
+  local redis_pub, ok_pub, err = redis_pool.acquire()
 
   if not ok_pub then
+    ngx.log(ngx.WARN, "[priority auths] couldn't connect to redis pub: ", err)
```

Comment:
It's OK for now, but later we might want to refactor this and extract parts of the logging functionality to its own module.